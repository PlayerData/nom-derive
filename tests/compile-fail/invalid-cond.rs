extern crate nom;
extern crate nom_derive;
use nom_derive::Nom;

#[derive(Nom)]
pub struct S {
    #[nom(Cond="true")]
    a: u32,
}

fn main() {}