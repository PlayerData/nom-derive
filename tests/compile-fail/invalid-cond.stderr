error: Nom-derive: a condition was given, but the type is not an option type. Hint: use Option<...>
 --> $DIR/invalid-cond.rs:8:8
  |
8 |     a: u32,
  |        ^^^
